moveos/moveos-stdlib/moveos-stdlib/sources/simple_multimap.move

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+/// A simple map that stores key/value pairs in a vector, and support multi values for one key.
+module moveos_std::simple_multimap {
+    use std::option;
+    use std::vector;
+
+   
+    /// Map key is not found
+    const ErrorKeyNotFound: u64 = 1;
+
+    struct SimpleMultiMap<Key, Value> has copy, drop, store {
+        data: vector<Element<Key, Value>>,
+    }
+
+    struct Element<Key, Value> has copy, drop, store {
+        key: Key,
+        value: vector<Value>,
```

Comment:
Allow duplicate values in vector?

Reply:
> Allow duplicate values in vector?

check repeat is expensive. the caller to decide if check repeat.